pywinauto/controls/uia_controls.py

```diff
@@ -356,7 +368,11 @@ def selected_index(self):
             return self.selected_item_index()
         except NoPatternInterfaceError:
             # workaround for Qt5 and WinForms
-            return self.texts().index(self.selected_text())
+            try:
+                return self.children(control_type='List')[0].texts().index(self.selected_text().splitlines())
+            except IndexError:
```

Comment:
The above line contains 2 places which can raise `IndexError`. It looks a little bit dangerous to catch both. Is it what you want?